src/components/pages/Subjects.vue

```diff
@@ -223,7 +223,67 @@
                           </v-card>
                         </v-dialog>
                       </v-list-item>
-
+                      <!-- Download archive -->
+                      <v-list-item link v-if="!item.trashed">
+                        <v-dialog
+                                v-model="showArchiveDialog"
+                                v-click-outside="clickOutsideDialogSubjectHideMenu"
+                                max-width="500">
+                          <template v-slot:activator="{ on }">
+                            <v-list-item-title v-on="on">Download data...</v-list-item-title>
+                          </template>
+                          <v-card>
+                            <v-card-text class="pt-4">
+                              <v-row class="m-0">
+                                <v-col cols="2">
+                                  <v-icon x-large color="green">mdi-download</v-icon>
+                                </v-col>
+                                <v-col cols="10">
+                                  <p v-if="isArchiveInProgress & !isArchiveDone">
+                                    <v-progress-circular  indeterminate class="mr-2" color="grey" size="14" width="2" />
+                                    Download in progress...
+                                  </p>
+                                  <p v-if="!(isArchiveInProgress || isArchiveDone)">
+                                    Do you want to download all data associated to the
+                                    session <code>{{item.name}}</code>?
+                                    (This includes every trial associated to it, and can take several minutes).
+                                  </p>
+                                  <p v-if="isArchiveDone">
+                                    The archive has been generated successfully!
```

Comment:
Rephrase: The archive has been successfully generated. Click on data.zip to download.